generators/app/templates/src/app/screens/RecoverPassword/index.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import renderer from 'react-test-renderer';
+
+import { RootComponent } from '~utils/tests';
+
+import RecoverPassword from './index';
+
+describe('#RecoverPassword', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = mount(
+      <RootComponent>
+        <RecoverPassword />
+      </RootComponent>
+    );
+  });
+
+  afterEach(() => {
+    component.unmount();
+  });
+
+  describe('when mounting the application', () => {
+    it('matches last snapshot', () => {
+      expect(renderer.create(component).toJSON()).toMatchSnapshot();
```

Comment:
Made a comment about this in this other pr  https://github.com/Wolox/react-bootstrap/pull/119#discussion_r405060028
Just to discuss, don't need to change it.